sql/mysqld.cc

```diff
@@ -8687,6 +8689,8 @@ SHOW_VAR status_vars[] = {
     {"Queries", (char *)&show_queries, SHOW_FUNC, SHOW_SCOPE_ALL},
     {"Questions", (char *)offsetof(System_status_var, questions),
      SHOW_LONGLONG_STATUS, SHOW_SCOPE_ALL},
+    {"Rbr_unsafe_queries", (char *)&rbr_unsafe_queries, SHOW_LONGLONG,
```

Comment:
Does this new status var need to be reflected in `perfschema.show_sanity` MTR test case?